class NotFoundException(Exception):
    def __init__(self):
        pass

class NotAvailableException(Exception):
    def __init__(self):
        pass